### PR TITLE
Fix Rails 4.2 Upgrade Issues

### PR DESCRIPTION
**Pull request by jhanggi**
This fixes a couple issues, primarily around POST/PUTs changing to GETs in the ajax searches.

It also begins to add some fixtures, which I would like to continue to use in feature specs. I'll have a few more fixtures as I track down the reservation creation stack overflow.

### CODE REVIEW

File: spec/fixtures/order_statuses.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+new_order_status:
+  name: New
```
Comment (drench) [inline spec/fixtures/order_statuses.rb]: Code Climate flagged this. Guessing it was meant to be `.yml` but is it being used?

**Comment by drench** [comment]
One question on a fixture but otherwise LGTM. Looking forward to more feature specs in this project.

For the other 2 RuboCop/CC complaints in `app/controllers/account_price_group_members_controller.rb` I'd run an autocorrect.

File: spec/fixtures/order_statuses.rb (hunk 1 above)
Comment (jhanggi) [inline spec/fixtures/order_statuses.rb]: Shouldn't be used. Removed it and we should see green specs.

**Comment by drench** [comment]
:+1: with green CI.

**Comment by jhanggi** [comment]
I also had a mistake in the feature filename. Didn't end in `_spec.rb`, so it wouldn't have run... dangerous.

**Pull request merged**